pkg/node/address.go

```diff
@@ -236,6 +237,16 @@ func GetIPv4Router() net.IP {
 	return ipv4RouterAddress
 }
 
+// SetExternalIP sets the external IP node address. It must be a public IP.
+func SetExternalIP(ip net.IP) {
```

Comment:
I think we should rename this to `SetK8sExternalIP` (same goes to `GetExternalIP`) to better reflect that this is k8s Node ExternalIP rather than a public facing iface IP addr?

Also, what happens in the DualStack setup? Is it possible to have different IP addrs for IPv4 and IPv6? /cc @fristonio

Reply:
Renamed it to `SetExternalIPv4` and `SetExternalIPv6`. I think in DualStack setup, it is possible to have different IP address for IPV4 and IPV6.